llvm/include/llvm/Support/TypeSize.h

```diff
@@ -244,6 +244,14 @@ template <typename LeafTy, typename ValueTy> class FixedOrScalableQuantity {
         isScalable());
   }
 
+  /// Returns a value X where the known coefficient will be rounded up to the
+  /// closest power-of-two, except for the zero coefficient that remains zero.
+  constexpr LeafTy coefficientPowerOf2Ceil() const {
```

Comment:
I may be missing something but it seems like the function is not used? Or is there some indirect use I did miss somehow?

Reply:
+1 (& if it is being added, probably add it separately with a unit test, then use it from MLIR 
 but, yeah, good to make sure there's a use for it first)

Reply:
I ended up not needing this in my last version, deleting.